code/modules/reagents/chemistry/reagents/medicine.dm

```diff
@@ -106,6 +106,59 @@
 		if(!Org.is_robotic())
 			Org.rejuvenate()
 
+/datum/reagent/medicine/tran_acid
+	name = "Tranexamic acid"
+	id = "tran_acid"
+	description = "A specialised chemical used for the treatment of truamatic bleeding and to control bleeding during surgery."
+	reagent_state = LIQUID
+	color = "#fcec96"
+	overdose_threshold = 40
+	harmless = FALSE
+	taste_description = "time running out"
+
+// This reagent's effects are handled in handle blood  code
+
+/datum/reagent/medicine/quikclot
```

Comment:
Having name and id be so different is way too confusing, and not worth it IMO.
I agree that `calcium_zeolite` is awkward to write, but you write code once, and read... a lot. Even you, the PR author, would likely get a "wait, what the heck is quickclot?" moment several months down the road.

Reply:
I was using the brand name of the mineral because it was fast and easy. I can change that to something else.